src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./Footer.css";
+
+export default function Footer() {
+    return (
+        <div className="footerContainer">
```

Comment:
not sure if you want to add a `TODO` here or something to remember to come back and add links later

Reply:
oh wait this is the footer section fomr before. You may need to merge updated main into this branch to get these changes from the last PR